pymess/backend/sms/__init__.py

```diff
@@ -101,8 +99,21 @@ def bulk_check_sms_states(self):
 
         if settings.SMS_SET_ERROR_TO_IDLE_MESSAGES:
             idle_output_sms.update(
-                state=self.model.STATE.ERROR, error=ugettext('timeouted')
+                state=self.model.STATE.ERROR, error=_('timeouted')
+            )
+
+    def _is_recipient_phone_number_allowed(self, message):
```

Comment:
so it can be safely tested on the server with other valid phone numbers

Reply:
I wanted to make one Generic config option for all SMS backends but in code it looks complicated :(

Reply:
I see you are using a generic option, or?

Reply:
maybe better solution is have dynamic backend configuration and for spicific number will be selected specific backend